libs/core/__deprecated__/renderer/components/core/icon/icon.config.ts

```diff
@@ -125,6 +125,7 @@ import Warning from "Core/__deprecated__/renderer/svg/warning.svg"
 import MarkAsUnread from "Core/__deprecated__/renderer/svg/mark-as-unread.svg"
 import Conversation from "Core/__deprecated__/renderer/svg/conversation.svg"
 import Exclamation from "Core/__deprecated__/renderer/svg/exclamation.svg"
+import DataMigration from "../../../../../../generic-view/ui/src/lib/icon/svg/data-migration.svg"
```

Comment:
Just to be sure. Is the import type used to prevent circular dependency?

Reply:
Unfortunately... yes.